packages/experimental/Spinner/SPEC.md

```diff
@@ -0,0 +1,162 @@
+# Spinner
+
+## Background
+
+The `Spinner` is an outline of a circle which animates around itself, to visually indicate that something is processing. Spinners are indeterminate progress indicators meaning that they are used when it is unknown how long a task will take to complete.
+
+### Best Practices
+
+- Use a Spinner when a task is not immediate
+- Use one Spinner at a time
+- Descriptive verbs are paired with a Spinner to help the user understand what is happening. Ie. Saving, processing, updating.
+- Use a Spinner when confirming a change has been made or a task is being processed
+
+## Sample Code
+
+### Basic Mobile Spinner
+
+```tsx
+<>
+  <Spinner /> // This is a spinner with default size as 'medium'
+  <Spinner size="x-small" /> // Spinner with a defined size
+  <Spinner status={status} /> // status property depicts it is animating or not
```

Comment:
Does this naming align with web api's for spinner? If not, will a name like "`isRotating`" be more readable?

Reply:
status is enum which has value as "active" and "unactive" which depicts the spinning which aligns with web api. 
I can change the comments though to "depicts it is rotating or not"